server/repositories/cmsQueries/categoryPageQuery.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable class-methods-use-this */
+const { DrupalJsonApiParams: Query } = require('drupal-jsonapi-params');
+const { getSmallTile, getLargeTile } = require('../../utils/jsonApi');
+
+class CategoryPageQuery {
+  static #TILE_FIELDS = [
+    'drupal_internal__nid',
+    'drupal_internal__tid',
+    'title',
+    'field_moj_description',
+    'field_moj_thumbnail_image',
+    'field_featured_image',
+    'field_moj_secondary_tags',
+    'path',
+  ];
+
+  constructor(establishmentName, uuid) {
+    this.establishmentName = establishmentName;
+    this.uuid = uuid;
+    this.query = new Query()
+      .addFields('node--page', CategoryPageQuery.#TILE_FIELDS)
+      .addFields('node--moj_video_item', CategoryPageQuery.#TILE_FIELDS)
+      .addFields('node--moj_radio_item', CategoryPageQuery.#TILE_FIELDS)
+      .addFields('moj_pdf_item', CategoryPageQuery.#TILE_FIELDS)
+      .addFields('taxonomy_term_series', CategoryPageQuery.#TILE_FIELDS)
+      .addInclude([
+        'field_featured_tiles',
+        'field_featured_tiles.field_moj_thumbnail_image',
+        'field_featured_tiles.field_featured_image',
+      ])
+      .addPageLimit(50)
```

Comment:
Think this should be removed just so it doesn't trip anyone up in the future. (There's only one item being returned in this JSON:API query, the category taxonomy term.  The `field_featured_tiles` are all part of that taxonomy term so are not subjected to a count).